house/views.py

```diff
@@ -20,7 +23,33 @@ def global_page(request):
 
 
 def house_login(request):
-    raise NotImplementedError
+
+    errorMsg = ""
+
+    # if this is a POST request we need to process the form data
+    if request.method == 'POST':
+        try:
+            # create a form instance and populate it with data from the request:
+            form = HouseIDForm(request.POST)
+            house_id = form.data["house_id"]
+
+            if House.objects.filter(house_id=house_id).count() == 1:
+                return HttpResponseRedirect(f'/../{house_id}')
+
+            else:  # count() == 0
+                errorMsg = "There is no House with the provided ID"
+                form = HouseIDForm()
+
+        # In case an exception not a valid UUID is thrown
```

Comment:
I think there are still too many comments.
“Good code is self documenting"

Reply:
I'll remove some